web/src/portico/team.ts

```diff
@@ -17,21 +40,24 @@ const repo_name_to_tab_name = {
 
 // Remember the loaded repositories so that HTML is not redundantly edited
 // if a user leaves and then revisits the same tab.
-const loaded_repos = [];
+const loaded_repos: string[] = [];
 
-function calculate_total_commits(contributor) {
+function calculate_total_commits(contributor: Contributor): number {
     let commits = 0;
-    for (const repo_name of Object.keys(repo_name_to_tab_name)) {
```

Comment:
We can get rid of the assertion by using `RepositoryName` here instead of `Object.keys(repo_name_to_tab_name)`.

Reply:
Thanks for that suggestion. Yeah you are right, I dunno how that didn't come across my mind.